sdk/search/Azure.Search.Documents/src/Options/SearchOptions.cs

```diff
@@ -201,26 +201,28 @@ internal string QueryAnswerRaw
         {
             get
             {
-                string queryAnswerStringValue = null;
-
                 if (QueryAnswer.HasValue)
                 {
-                    var queryAnswerValues = new List<string>();
+                    StringBuilder queryAnswerStringValue = new($"{QueryAnswer.Value}");
```

Comment:
The compiler might elide this, but it's wasteful if not. If `QueryAnswer.Value` is already a string, there is no need. If it's not, `QueryAnswer.Value?.ToString()` is faster. Formattable strings like this incur an allocation to call into a long chain of formatting type queries that fall back to just calling `ToString()`. You can cut out all that.